articles/connections/database/custom-db/lock.md

```diff
@@ -0,0 +1,11 @@
+---
+description: Learn how to use Lock with your custom database.
+crews: crew-2
+---
+# Using Lock with Custom Databases
+
+If you use [Lock](libraries#lock), enabling the database connection lets users enter their username and password on the Auth0 Login widget. Once entered, this data is passed to your scripts.
+
+![Auth0 login widget](/media/articles/connections/database/mysql/db-connection-widget.png)
+
+<%= include('../../_quickstart-links.md') %>
```

Comment:
I don't see the logical connection between this doc and the next steps listed here